addons/hr_expense/models/hr_expense.py

```diff
@@ -303,7 +303,7 @@ def _create_sheet_from_expenses(self):
             'name': todo[0].name if len(todo) == 1 else '',
             'expense_line_ids': [(6, 0, todo.ids)]
         })
-        sheet._onchange_employee_id()
+        sheet._compute_from_employee_id
```

Comment:
As onchanges are only executed on the interface, it was then mandatory to call it on a python code.

Here, the method _compute_from_employee_id should be called right ?

So isn't it enough to just drop the line ?

Reply:
Whoops. Yep, you're right